cmd/atlas/internal/client/schema.go

```diff
@@ -0,0 +1,15 @@
+package client
+
+import (
+	"context"
+)
+
+// Inspect returns the schema description by its name.
+func Inspect(ctx context.Context, d Driver) ([]byte, error) {
+	return nil, nil
+}
```

Comment:
these types should be associated with your struct. 
also if they return []byte, you must specify a `schemaspec.Marshaler` to specify under which syntax the transformation to bytes should happen.